pkg/vm/engine/tae/db/base_test.go

```diff
@@ -187,6 +188,12 @@ func (e *testEngine) truncate() {
 
 func initDB(t *testing.T, opts *options.Options) *DB {
 	dir := testutils.InitTestEnv(ModuleName, t)
+	if os.Getenv("skipSync") == "true" {
```

Comment:
Wait.   Do you want to use env?    Should this be in config?   Or if it is ut, why not just hard code it.   Better, have two UT test both true and false. 

 It is better for UT to not rely on anything else.